src/generic/splitter.cpp

```diff
@@ -871,6 +880,12 @@ bool wxSplitterWindow::Unsplit(wxWindow *toRemove)
         return false;
     }
 
+    if (m_splitMode == wxSPLIT_VERTICAL) {
+        m_latestVerticalSashPosition = m_sashPosition;
+    } else if (m_splitMode == wxSPLIT_HORIZONTAL) {
+        m_latestHorizontalSashPosition = m_sashPosition;
+    }
+
```

Comment:
Same style comment as above:
```suggestion
    if (m_splitMode == wxSPLIT_VERTICAL)
    {
        m_lastSplitPosition.y = m_sashPosition;
    }
    else if (m_splitMode == wxSPLIT_HORIZONTAL)
    {
        m_lastSplitPosition.x = m_sashPosition;
    }

```